.ci/Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library('corda-shared-build-pipeline-steps@5.0') _
+@Library('corda-shared-build-pipeline-steps@chrisbarratt/CORE-8075-create-pipeline-for-running-the-autotester') _

```

Comment:
I presume this was just for testing and you'll want to change prior to merge ?

is there also a associated corda-shared-build-pipeline-steps PR to merge this into 5.0 branch?